server/ansible/roles/pbench-server-activate-setup-web-server/tasks/main.yml

```diff
@@ -0,0 +1,14 @@
+---
+- name: "make sure rsync is installed"
+  package:
+    name: rsync
+    state: latest
+
+# This needs passwordless login (as root) from a node to *itself* (*not* localhost)
```

Comment:
Why do we need this?  There is no way to do this locally without ssh?

Reply:
I'm not sure why I thought this was necessary. I retested and I believe it  is not needed.